kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx

```diff
@@ -54,4 +74,21 @@ describe('Overview', () => {
       expect(componentEmpty.find('td').text()).toEqual('No Partitions found');
     });
   });
+
+  describe('should render circular alert', () => {
+    it('should be in document', () => {
+      renderComponent();
+      const circles = screen.getAllByRole('circle');
+      expect(circles[0]).toBeInTheDocument();
```

Comment:
To my mind 82-83 rows are unnecessary, because on row 84, you check the same thing

Reply:
Yes, you are right. I will delete.